<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Join</class>
 <widget class="QDialog" name="Join">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NSUCircle - 회원가입</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;나눔바른고딕&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;맑은 고딕&quot;;</string>
       </property>
       <property name="title">
        <string>회원가입</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <spacer name="horizontalSpacer_6">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>30</width>
                      <height>17</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLabel" name="joinStuNumLabel">
                    <property name="styleSheet">
                     <string notr="true">font: 10pt &quot;맑은 고딕&quot;;</string>
                    </property>
                    <property name="text">
                     <string>학번</string>
                    </property>
                    <property name="buddy">
                     <cstring>joinstuNumLineEdit</cstring>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>85</width>
                    <height>13</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <spacer name="horizontalSpacer_5">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>17</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLabel" name="joinIDLabel">
                    <property name="styleSheet">
                     <string notr="true">font: 10pt &quot;맑은 고딕&quot;;</string>
                    </property>
                    <property name="text">
                     <string>아이디</string>
                    </property>
                    <property name="buddy">
                     <cstring>joinIDLineEdit</cstring>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer_13">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>85</width>
                    <height>13</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <spacer name="horizontalSpacer_4">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>13</width>
                      <height>17</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLabel" name="joinPasswordLabel">
                    <property name="styleSheet">
                     <string notr="true">font: 10pt &quot;맑은 고딕&quot;;</string>
                    </property>
                    <property name="text">
                     <string>비밀번호</string>
                    </property>
                    <property name="buddy">
                     <cstring>joinPasswordLineEdit</cstring>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer_14">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>85</width>
                    <height>13</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <spacer name="horizontalSpacer_11">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>10</width>
                      <height>14</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLabel" name="joinPasswordLabel_2">
                    <property name="styleSheet">
                     <string notr="true">font: 10pt &quot;맑은 고딕&quot;;</string>
                    </property>
                    <property name="text">
                     <string>비밀번호 확인</string>
                    </property>
                    <property name="buddy">
                     <cstring>joinPasswordLineEdit_2</cstring>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QLineEdit" name="joinstuNumLineEdit"/>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>17</width>
                    <height>28</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLineEdit" name="joinIDLineEdit"/>
                </item>
                <item>
                 <spacer name="verticalSpacer_6">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>17</width>
                    <height>28</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLineEdit" name="joinPasswordLineEdit"/>
                </item>
                <item>
                 <spacer name="verticalSpacer_8">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>17</width>
                    <height>28</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLineEdit" name="joinPasswordLineEdit_2"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>13</width>
                <height>13</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <spacer name="horizontalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Expanding</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>13</width>
                      <height>17</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLabel" name="joinNameLabel">
                    <property name="styleSheet">
                     <string notr="true">font: 10pt &quot;맑은 고딕&quot;;</string>
                    </property>
                    <property name="text">
                     <string>이름</string>
                    </property>
                    <property name="buddy">
                     <cstring>joinNameLineEdit</cstring>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer_11">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>53</width>
                    <height>13</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <spacer name="horizontalSpacer_8">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Expanding</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>13</width>
                      <height>17</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLabel" name="joinDepartmentLabel">
                    <property name="styleSheet">
                     <string notr="true">font: 10pt &quot;맑은 고딕&quot;;</string>
                    </property>
                    <property name="text">
                     <string>소속학과</string>
                    </property>
                    <property name="buddy">
                     <cstring>joinDepartLineEdit</cstring>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer_12">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>53</width>
                    <height>13</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Expanding</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>13</width>
                      <height>16</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLabel" name="joinHobbyLabel">
                    <property name="styleSheet">
                     <string notr="true">font: 10pt &quot;맑은 고딕&quot;;</string>
                    </property>
                    <property name="text">
                     <string>취미</string>
                    </property>
                    <property name="buddy">
                     <cstring>joinHobbyLineEdit</cstring>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer_10">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>53</width>
                    <height>13</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Expanding</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>10</width>
                      <height>13</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLabel" name="joinPhoneNumLabel">
                    <property name="styleSheet">
                     <string notr="true">font: 10pt &quot;맑은 고딕&quot;;</string>
                    </property>
                    <property name="text">
                     <string>전화번호</string>
                    </property>
                    <property name="buddy">
                     <cstring>joinPhoneNumLineEdit</cstring>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLineEdit" name="joinNameLineEdit"/>
                </item>
                <item>
                 <spacer name="verticalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>21</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLineEdit" name="joinDepartLineEdit"/>
                </item>
                <item>
                 <spacer name="verticalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>22</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLineEdit" name="joinHobbyLineEdit"/>
                </item>
                <item>
                 <spacer name="verticalSpacer_7">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>18</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLineEdit" name="joinPhoneNumLineEdit"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>15</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <spacer name="horizontalSpacer_12">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>338</width>
                <height>28</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QComboBox" name="translateComboBox">
                <property name="minimumSize">
                 <size>
                  <width>101</width>
                  <height>31</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>101</width>
                  <height>31</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 10pt &quot;맑은 고딕&quot;;</string>
                </property>
                <item>
                 <property name="text">
                  <string>한국어</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>English</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="translateButton">
                <property name="minimumSize">
                 <size>
                  <width>81</width>
                  <height>31</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>81</width>
                  <height>31</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 10pt &quot;맑은 고딕&quot;;</string>
                </property>
                <property name="text">
                 <string>번역</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_9">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>3</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>101</width>
           <height>24</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="joinOkButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>91</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>91</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;맑은 고딕&quot;;</string>
         </property>
         <property name="text">
          <string>확인</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="joinCancelButton">
         <property name="minimumSize">
          <size>
           <width>91</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>91</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;맑은 고딕&quot;;</string>
         </property>
         <property name="text">
          <string>취소</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>101</width>
           <height>24</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>joinstuNumLineEdit</tabstop>
  <tabstop>joinIDLineEdit</tabstop>
  <tabstop>joinPasswordLineEdit</tabstop>
  <tabstop>joinPasswordLineEdit_2</tabstop>
  <tabstop>joinNameLineEdit</tabstop>
  <tabstop>joinDepartLineEdit</tabstop>
  <tabstop>joinHobbyLineEdit</tabstop>
  <tabstop>joinPhoneNumLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
